### ISSUE TITLE
Fractions miss-behaving when next to other numbers

### ISSUE THREAD

**Issue by Jakeii** [spencermountain/compromise#792]
This works as expected (I think):
`nlp('10 1/2').numbers().json()` or `nlp('10 1/2').fractions().json()`

```json
[{
    "text": "1 1/2",
    "normal": "1 1/2",
    ...
    "prefix": "",
    "number": 1.5,
    "suffix": "",
    "cardinal": "1.5",
    "ordinal": "1.5th",
}]
```

But this doesn't 
`nlp('1/2 10').numbers().json()` or `nlp('1/2 10').fractions().json()`
```.json
[
  {
    "text": "1/2 10",
    "normal": "1/2 10",
    ...
    "prefix": "",
    "number": 10.5,
    "suffix": "",
    "cardinal": "10.5",
    "ordinal": "10.5th"
  }
]
```

or this weirdness:
`nlp('2 1/2 30').numbers().json()`
```json
[
  {
    "text": "10 1/2 10",
    "normal": "10 1/2 10",
    ...
    "prefix": "",
    "number": 10.5,
    "suffix": "",
    "cardinal": "10.5",
    "ordinal": "10.5th",
  }
]
```

A real world example where I'm trying to use this is for something like `2 1/2 30 gram packets of food`

**Comment by spencermountain** [comment]
hi Jake, good catch. The 'and a half' stuff is the weakest part of number-parsing, and these are good examples.
Happy to take a look at them. If anyone else is curious, the code in question [is here](https://github.com/spencermountain/compromise/tree/master/plugins/numbers/src/convert/toNumber).

The `2 1/2 30` issue seems like a number tokenization issue that will need a condition [somewhere here](https://github.com/spencermountain/compromise/blob/master/plugins/numbers/src/find.js) 
thanks

**Comment by Jakeii** [comment]
@spencermountain thanks for the fast response and clues, I'll see about doing a PR.

**Comment by Jakeii** [comment]
@spencermountain I see fractions aren't tagged with `#NumericValue` is there a reason for that?

if I do:
```
  world.postProcess((doc) => {
    doc.match('#Fraction').tag('#NumericValue');
  });
```
'2 1/2 30' and '1/2 10' are split up correctly, then could add a case to the `toNumber` function you linked to for `and a half` stuff.

**Comment by spencermountain** [comment]
will ping you when this is released - hopefully this week.
thanks

**Comment by spencermountain** [comment]
released as `13.8.0`
thanks for your help!
